cmd/bbolt/main.go

```diff
@@ -1292,7 +1308,9 @@ func (cmd *benchCommand) runWritesNestedWithSource(db *bolt.DB, options *BenchOp
 				if err := b.Put(key, value); err != nil {
 					return err
 				}
-
+				if *keys != nil {
+					*keys = append(*keys, nestedKey{nil, key})
```

Comment:
If you run `./bbolt bench -write-mode seq-nest -read-mode rnd`, you will see the `bench: read: read seq-nest: iter mismatch: expected 1000, got 0` error.

```suggestion
					*keys = append(*keys, nestedKey{
						bucket: name,
						key:    key,
					})					
```